src/openvic-simulation/misc/Define.cpp

```diff
@@ -150,15 +150,25 @@ bool DefineManager::load_define_years(Timespan& value, Define::Type type, std::s
 DefineManager::DefineManager()
   : // Date
 	start_date { 1836, 1, 1 },
-	end_date { 1936, 1, 1 }
+	end_date { 1936, 1, 1 },
 
 	// Country
+	great_power_rank { 8 },
+	lose_great_power_grace_days { Timespan::from_years(1) },
+	secondary_power_rank { 16 },
+	country_investment_industrial_score_factor { 1 },
 
 	// Economy
 
 	// Military
+	pop_size_per_regiment { 1000 },
```

Comment:
3000 in vanilla. Not sure if you want to set everything to vanilla since it will be overwritten anyway...

Reply:
I set arbitrary numbers just to have something there by default, the player can load vanilla numbers from their own copy of the base game if they wish, we don't provide that data ourselves.